### ISSUE TITLE
Does anyone have a template for A340-300? Can you send it to me

### ISSUE THREAD

**Issue by HO1076** [kolos26/GEOFS-LiverySelector#804]

**Comment by CaptainAshAdult** [comment]
> _No description provided._

Same Here!!!! I Really Need It!

**Comment by Bilibilizm** [comment]
A340-300map.png

**Comment by CaptainAshAdult** [comment]
> A340-300map.png

Umm Where Ist The Map?

**Comment by aviator200** [comment]
How come no image you sent there??

**Comment by SolAva8** [comment]
![IMG_6472](https://github.com/user-attachments/assets/89911aca-a66f-400b-90c7-b7fda4c709e1)

**Comment by SolAva8** [comment]
But join the discord because we already made pretty all the a340 operators liveries. There isn’t much left.

**Comment by Bilibilizm** [comment]
> But join the discord because we already made pretty all the a340 operators liveries. There isn’t much left.

Yes bro,so someone who uses github they cant do more liveries for

**Comment by HO1076** [comment]
> ![IMG_6472](https://private-user-images.githubusercontent.com/129607542/356765289-89911aca-a66f-400b-90c7-b7fda4c709e1.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMyODI2MTYsIm5iZiI6MTcyMzI4MjMxNiwicGF0aCI6Ii8xMjk2MDc1NDIvMzU2NzY1Mjg5LTg5OTExYWNhLWE2NmYtNDAwYi05MGM3LWI3ZmRhNGM3MDllMS5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgxMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MTBUMDkzMTU2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTYwMmFjMWI0MWQzMTRjZDEzNzkyMWU3ZGVjYWY5MTJjMDIxMWQ4YTU0ZGE3NTJhMDc5ODE3NzIzYzI4NWRkMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.hfKcbVqtDjmhgI4ERIjlpojjE5G3jwK-kuxxm6Z2Tjc)

Thank you very much
